src/components/tool/guide/Config/ConfigDuration.stories.js

```diff
@@ -0,0 +1,86 @@
+import ConfigDuration from "./ConfigDuration";
+import {useState} from "react";
+import {useArgs} from "@storybook/preview-api";
+
+export default {
+    title: 'Tools/Config/ConfigDuration',
+    component: ConfigDuration,
+    decorators: [
+        (Story) => {
+            const [args, setArgs] = useArgs();
+            const onChange = (value) => {
+                setArgs({value});
+            };
+            return <Story args={{...args, onChange}}/>;
+        },
+    ]
+}
+
+// Default story
+export const Default = {
+    args: {
+        title: 'Duration',
+        description: "Tooltip example",
+        selected: false,
+        value: { hour: 1, minute: 30 },
+        inputPropsHour: { min: 1, max: 167,  step: 1 },
+        inputPropsMinute: { min: 0, max: 59,  step: 1 },
+    }
+}
+
+// Duration Component with Alert Dismissible
+export const AlertDismissible = {
+    args: {
+        title: 'Duration Alert',
+        description: "Second tooltip example",
+        selected: true,
+        value: { hour: 100, minute: 30 },
+        inputPropsHour: { min: 1, max: 167,  step: 1 },
+        inputPropsMinute: { min: 0, max: 59,  step: 1 },
+        showAlert: true,
+        alertDismissible: true,
+        alertType: "warning",
+        alertMsg: "This is a dismissible alert",
+    }
+}
+
+// Duration Component with Alert Dismissible
```

Comment:
Comments are redundant, and this one is incorrect. Should be removed

Reply:
Fixed